02-spatial-data.Rmd

```diff
@@ -652,10 +660,11 @@ extent(new_raster)
 crs(new_raster)
 ```
 
-Unlike the **sf** package, **raster** only use the `proj4string` representation of the coordinate reference system.
+<!--CRSargs(CRS("+init=epsg:4326"))-->
+Note that in contrast to the **sf** package, **raster** only accepts the `proj4string` representation of the coordinate reference system.
 
 Values of the cells are the only non-spatial information in the `Raster*` object.
```

Comment:
My point was that `+init=epsg:4326` is the proj4string representation, while `4326` is just a number (epsg representation). You can have both in sf and only a first one in raster.